frontend/src/Components/MyHistory/SingleDayTasksContainer.tsx

```diff
@@ -8,20 +8,33 @@ const SingleDayTasksContainer = (props: {
   historyTasks: TaskInterface[];
 }) => {
   const filteredTasks = props.historyTasks.filter(
-    (task) => task.deliveryTime.toDateString() === props.date.toDateString()
+    (task) => task.date.toDateString() === props.date.toDateString()
   );
 
   return (
-    <Box sx={{display: "flex", flexDirection: "column"}}>
+    <Box sx={{ display: "flex", flexDirection: "column" }}>
       <HistoryTaskDate date={props.date} />
       {filteredTasks.map((task) => (
-        <Box key={task.id} sx={{ display: "flex", mb: 1, p: 1, bgcolor: "#FFFFFF", border: "1px solid #DFDFDF", borderRadius: 2, flexDirection: "column", justifyContent: "flex-start" }}>
-          <Typography sx={{ mr: 1 }}>{task.name}</Typography>
+        <Box
+          key={task.id}
+          sx={{
+            display: "flex",
+            mb: 1,
+            p: 1,
+            bgcolor: "#FFFFFF",
+            border: "1px solid #DFDFDF",
+            borderRadius: 2,
+            flexDirection: "column",
+            justifyContent: "flex-start",
+          }}
+        >
+          <Typography sx={{ mr: 1 }}>client name</Typography>
           <Typography variant="caption">
-            {"Delivered at " + task.deliveryTime.toLocaleTimeString([], {
-              hour: "2-digit",
-              minute: "2-digit",
-            })}
+            {"Delivered at " +
+              task.date.toLocaleTimeString([], {
```

Comment:
this doesn't seem to show up?

Reply:
yeah left it there just in case but now deleted!